frontend/src/lib/utils/eventUsageLogic.ts

```diff
@@ -43,7 +43,7 @@ export const eventUsageLogic = kea<eventUsageLogicType>({
                 posthog.capture('annotation viewed', properties)
             }
         },
-        reportPersonDetailViewed: async ({ person }, breakpoint) => {
+        reportPersonDetailViewed: async ({ person }: { person: PersonType }, breakpoint) => {
```

Comment:
I think the issue here comes from `actions:` on lines 13 and below not being typed. We should type things there as well.